ra_aid/tools/programmer.py

```
@@ -1,42 +1,51 @@
 from ra_aid.proc.interactive import run_interactive_command
 from ra_aid.text.processing import truncate_output
+from ra_aid.tools.memory import _global_memory
 
 console = Console()
 logger = get_logger(__name__)
 
+
 @tool
-def run_programming_task(instructions: str, files: List[str] = []) -> Dict[str, Union[str, int, bool]]:
+def run_programming_task(
+    instructions: str, files: List[str] = []
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)